src/main/webapp/app/exam/participate/exam-cover/exam-participation-cover.component.html

```diff
@@ -109,19 +109,87 @@ <h2 class="col-md-6" style="text-align: center; font-weight: normal">
                     </div>
                 </div>
             </div>
-            @if (startView) {
-                <ng-container #startButton>
-                    @if (!startButtonEnabled) {
-                        <div>{{ 'artemisApp.exam.startExamToolTip' | artemisTranslate }}</div>
+            <ng-container #startButton>
+                <button id="start-exam" [disabled]="!startButtonEnabled || waitingForExamStart" type="submit" (click)="startExam()" class="btn btn-primary">
+                    <span jhiTranslate="artemisApp.exam.startExam"></span>
+                </button>
+                @if (!startButtonEnabled) {
+                    <div class="mt-2 mb-2" style="font-weight: lighter">{{ 'artemisApp.exam.startExamToolTip' | artemisTranslate }}</div>
+                }
+            </ng-container>
+            @if (waitingForExamStart) {
+                <div class="exam-waiting-for-start-overlay alert alert-info">
+                    <span>{{ 'artemisApp.examParticipation.waitForStart' | artemisTranslate: { title: exam.title } }}</span>
```

Comment:
@pzdr7 Suggested to use a span with jhiTranslate. In order to make it consistent, could we use it here as well?